packages/r2frida/build.sh

```diff
@@ -0,0 +1,36 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/nowsecure/r2frida
+TERMUX_PKG_DESCRIPTION="Frida IO plugin for radare2"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="5.8.6"
+TERMUX_PKG_SRCURL=https://github.com/nowsecure/r2frida/archive/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=c65979581dc514e58639b5c9492ed21780370446fad6d8cfcaaffe25e86366f1
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_DEPENDS="radare2"
+TERMUX_PKG_BUILD_DEPENDS="radare2"
+
+termux_step_pre_configure () {
+	if [ $TERMUX_ARCH != "aarch64" ]; then
+		termux_error_exit "Unsupported arch: $TERMUX_ARCH"
```

Comment:
Set `TERMUX_PKG_BLACKLISTED_ARCHES="arm, i686, x86_64"` instead, if only aarch64 is supported

Reply:
Yes, right now r2frida only builds on android-arm64, but i wil add arm later when i get this to work (need to pull the SDK from frida which is arch-os specific)

Reply:
thanks. i used this because i saw other packages doing it:

```
$ git grep 'mux_error_exit "Unsupported arch: ' ..
../ecl/build.sh:                *)       termux_error_exit "Unsupported arch: $TERMUX_ARCH" ;;
../ffmpeg/build.sh:             termux_error_exit "Unsupported arch: $TERMUX_ARCH"
../libvpx/build.sh:             termux_error_exit "Unsupported arch: $TERMUX_ARCH"
../pandoc/build.sh:                     termux_error_exit "Unsupported arch: $TERMUX_ARCH"
```